Libraries/LibWeb/HTML/StructuredSerialize.cpp

```diff
@@ -776,8 +794,32 @@ class Deserializer {
             value = TRY(deserialize_reg_exp_object(*m_vm.current_realm(), m_serialized, m_position));
             break;
         }
-        // FIXME: 12. Otherwise, if serialized.[[Type]] is "SharedArrayBuffer", then:
-        // FIXME: 13. Otherwise, if serialized.[[Type]] is "GrowableSharedArrayBuffer", then:
+        // 12. Otherwise, if serialized.[[Type]] is "SharedArrayBuffer", then:
+        case ValueTag::SharedArrayBuffer: {
+            auto* realm = m_vm.current_realm();
```

Comment:
feels like we're missing some spec comments here

Reply:
Will add the spec in every case, thanks! :)

Reply:
Fixed.